Jenkinsfile

```diff
@@ -112,17 +203,24 @@ pipeline {
                 }
             }
         }
-        stage('Deploy JupyterHub to NCATS') {
-            agent {
-                node { label 'ls-api-ci.ncats' }
-            }
+        stage('Deploy JupyterHub to Kubernetes') {
```

Comment:
It looks like you replaced the deployment to Polus2 with another iteration of the deployment to AWS CI. There should be two deploy stages, one to `AWS CI` and the other to `NCATS`

Reply:
My mistake, fixed now